src/plugins/themes.js

```diff
@@ -0,0 +1,84 @@
+"use strict";
+
+const fs = require("fs");
+const Helper = require("../helper");
+const colors = require("colors/safe");
+const path = require("path");
+const themes = new Map();
+
+module.exports = {
+	get: get,
+	fileName: fileName
+};
+
+fs.readdir("client/themes/", (err, builtInThemes) => {
+	if (err) {
+		return;
+	}
+	builtInThemes
+		.filter((theme) => theme.endsWith(".css"))
+		.map(makeLocalThemeObject)
+		.forEach((theme) => themes.set(theme.name, theme));
+});
+
+fs.readdir(Helper.getPackagesPath(), (err, packages) => {
+	if (err) {
+		return;
+	}
+	packages
+		.map(makePackageThemeObject)
+		.forEach((theme) => {
+			if (theme) {
+				themes.set(theme.name, theme);
+			}
+		});
+});
+
+function get() {
```

Comment:
Nitpick but maybe better with `list` or `listAllThemes` or something like that? Don't worry, I'm not bikeshedding, I need to test this PR anyway when I have a minute :)

Reply:
Why? Is it because it's a map internally? 

1. The outer code doesn't care how it is stored internally, and shouldn't. 
2. If you want us to say it's an Array (it's not a list...) then let's do that in JS doc, not in the name of the function.
3. This is a getter function, get is a pretty standard name
4. If you want "all", then that's fine. I'd be happy with "getAll"

tl;dr, I'd be happy with "get" or "getAll", or I'd be happy to add jsdoc to make it clear it's an array it returns. Other than that, I'm going to push back.

Reply:
> Why? Is it because it's a map internally?

No no sorry, not at all. It's just that only `get()` seems not very descriptive enough (even for a getter), but I don't mind much anyway, that won't prevent me to approve once I can test. It's just, if there is a more descriptive name to give it, good, otherwise oh well.

Reply:
I mean, the module is "themes" and the function is "get". Seems pretty descriptive to me. As I said, happy to change to getAll if you would prefer that.

Reply:
I'm renaming this to getAll to make it clear that it gets ... all the themes.